Source/Params/param_includes/pelec_defaults.H

```diff
@@ -35,6 +35,7 @@ int PeleC::ppm_reference_eigenvectors = 0;
 int PeleC::plm_iorder = 2;
 int PeleC::hybrid_riemann = 0;
 int PeleC::use_colglaz = -1;
+int PeleC::use_laxf_flux = 0;
 int PeleC::riemann_solver = 0;
 int PeleC::cg_maxiter = 12;
```

Comment:
Unrelated but it seems like there's a bunch of stuff in here for deprecated Riemann solvers that might be able to be cleaned up

Reply:
I think we will add it in one day :)